edx-platform/pearson-certiport-theme/lms/templates/header/user_dropdown.html

```diff
@@ -3,16 +3,15 @@
 <%namespace name='static' file='static_content.html'/>
 
 <%!
-import json
-
 from django.conf import settings
 from django.urls import reverse
-from django.utils.translation import gettext as _
+from django.utils.translation import ugettext as _
 
 from openedx.core.djangoapps.user_api.accounts.image_helpers import get_profile_image_urls_for_user
 from openedx.core.djangoapps.user_api.accounts.toggles import should_redirect_to_order_history_microfrontend
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
-from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name, get_enterprise_learner_portal
+from openedx.features.enterprise_support.utils import get_enterprise_learner_generic_name
```

Comment:
Are these changes part of the Olive template or are the Certiport overwrites? @sergivalero20

Reply:
@Squirrel18  I want to remove this template since it has not custom changes.